src/sentry/tagstore/legacy/backend.py

```diff
@@ -110,6 +125,34 @@ def get_tag_values(self, project_ids, key, values=None):
 
         return list(qs)
 
+    def get_group_tag_key(self, group_id, key):
+        from sentry.tagstore.exceptions import GroupTagKeyNotFound
+
+        try:
+            return GroupTagKey.objects.get(
+                group_id=group_id,
+                key=key,
+            )
+        except GroupTagKey.DoesNotExist:
+            raise GroupTagKeyNotFound
+
+    def get_group_tag_keys(self, group_ids, keys=None, limit=None):
+        if isinstance(group_ids, list):
```

Comment:
Change this to `isinstance(group_ids, collections.Iterable)` just so we don't get bit when someone uses a tuple or a set.

Or, alternative, check for `isinstance(group_ids, six.integer_types)`